README.md

```diff
@@ -1,25 +1,43 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Your README file still contains empty headers from the `template`. Please either fill in the contents or get rid of them if they are not necessary. This will render our README file descriptive and professional. `Prerequisites, Setup, Install, Usage, Run tests Deployments` are empty.